gallery/src/pages/Accessibility.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxH3, NxList, NxTextLink } from '@sonatype/react-shared-components';
+
+const Accessibility = () =>
+  <GalleryTile title="Accessibility">
+    <NxP>
+      Accessibility (A11y) means ensuring that our applications are usable by everybody. We do this in a variety of
+      different ways. We make sure that our designs are clear and that they follow accessibility guidelines for
+      contrast and readability. Our components are written with semantic HTML, correct use of <NxCode>aria</NxCode>
+      tags, and testing in VoiceOver and ChromeVox accessibility is built into the Shared Components. However there is
+      still work that needs to be done to make sure that the application as a whole is accessible.
+    </NxP>
+
+    <NxH3 className="nx-tile__section-header">Making your pages accessible</NxH3>
+    <NxP>
+      Ensuring you use the components in an accessible manner is important, here are a few tips to help you make your
+      pages accessible.
+    </NxP>
+
+    <NxList className="nx-list--bulleted">
+      <NxList.Item>Tab order is important. Make sure that the tab order of the page is logical.</NxList.Item>
+      <NxList.Item>Related to the above, does the keyboard focus order follow the visual layout?</NxList.Item>
```

Comment:
How is this bullet different from the previous one?

Reply:
> How is this bullet different from the previous one?

https://github.com/sonatype/sonatype-react-shared-components/pull/455/commits/255eb871aebffa3f8037af54c431a8864ebc967b